docs/mobile-setup.md

```diff
@@ -7,65 +7,9 @@ does not yet officially support or run its own tests against a particular config
 Setting Up Protractor with Appium - Android/Chrome
 -------------------------------------
 ###### Setup
-*   Install Java SDK (>1.6) and configure JAVA_HOME (Important: make sure it's not pointing to JRE).
-*   Follow http://spring.io/guides/gs/android/ to install and set up Android developer environment. Do not set up Android Virtual Device as instructed here.
-*   From commandline, ```android avd``` and then install an AVD, taking note of the following:
-   * Start with an ARM ABI
-   * Enable hardware keyboard: ```hw.keyboard=yes```
-   * Enable hardware battery: ```hw.battery=yes```
-   * Use the Host GPU
-   * Here's an example:
-
-Phone:
-```shell
-> android list avd
-Available Android Virtual Devices:
-    Name: LatestAndroid
-  Device: Nexus 5 (Google)
-    Path: /Users/hankduan/.android/avd/LatestAndroid.avd
-  Target: Android 4.4.2 (API level 19)
- Tag/ABI: default/armeabi-v7a
-    Skin: HVGA
-```
 
-Tablet:
-```shell
-> android list avd
-Available Android Virtual Devices:
-    Name: LatestTablet
-  Device: Nexus 10 (Google)
-    Path: /Users/hankduan/.android/avd/LatestTablet.avd
-  Target: Android 4.4.2 (API level 19)
- Tag/ABI: default/armeabi-v7a
-    Skin: WXGA800-7in
-```
-*   Follow http://ant.apache.org/manual/index.html to install ant and set up the environment.
-*   Follow http://maven.apache.org/download.cgi to install mvn (Maven) and set up the environment. 
-   * NOTE: Appium suggests installing Maven 3.0.5 (I haven't tried later versions, but 3.0.5 works for sure).
-*   Install Appium using node ```npm install -g appium```. Make sure you don't install as sudo or else Appium will complain.
-   * You can do this either if you installed node without sudo, or you can chown the global node_modules lib and bin directories.
-*   Start emulator manually (at least the first time) and unlock screen.
-
-```shell
-> emulator -avd LatestAndroid
-```
-* Your devices should show up under adb now:
-
-```shell
-> adb devices
-List of devices attached 
-emulator-5554 device
-```
-*   If the AVD does not have chrome (and it probably won't if it just created), you need to install it:
-   * You can get v34.0.1847.114 from http://www.apk4fun.com/apk/1192/
-   * Once you download the apk, install to your AVD as such:
-
-```shell
-> adb install ~/Desktop/chrome-browser-google-34.0.1847.114-www.apk4fun.com.apk 
-2323 KB/s (30024100 bytes in 12.617s)
-Success
-```
-* If you check your AVD now, it should have Chrome.
+Use `webdriver-manager` to install `appium` and the Android SDK.  See details
+[here](https://github.com/angular/webdriver-manager/blob/master/mobile.md).
```

Comment:
Documentation nit: prefer useful descriptions in links, e.g. `See details [on the WebDriverManager page]` (a Tech Writer drilled this into my head a couple years ago so now I have to comment on it).